src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/IFileSystemMock.cs

```diff
@@ -205,28 +203,32 @@ public void WriteAllText(string path, string content)
 
         public void WriteAllText(string path, string content, Encoding encoding)
         {
-            if (_files.TryGetValue(path, out FileData curData))
+            if (Files.TryGetValue(path, out FileData data))
             {
                 // This makes sure each write to the file increases the timestamp
-                curData.FileContents = content;
-                curData.FileEncoding = encoding;
-                SetLastWriteTime(curData);
+                data.FileContents = content;
+                data.FileEncoding = encoding;
+                data.SetLastWriteTime();
             }
             else
             {
-                _files[path] = new FileData() { FileContents = content, FileEncoding = encoding };
-                SetLastWriteTime(_files[path]);
+                Files[path] = new FileData
+                {
+                    FileContents = content,
+                    FileEncoding = encoding,
+                    LastWriteTimeUtc = DateTime.UtcNow
```

Comment:
The call to `SetLastWriteTime` wasn't important?

Reply:
On a new instance it is equivalent.

Reply:
Great!